adsingestp/parsers/jats.py

```diff
@@ -190,24 +191,58 @@ def parse(self, article_metadata):
         # JATS puts author data in <contrib-group>, giving individual authors in each <contrib>
         for art_group in art_contrib_groups:
             art_contrib_group = art_group.extract()
-            contribs_raw = art_contrib_group.find_all("contrib")
+
+            # extract <contrib> from each <contrib-group>
+            contribs_raw_init = []
+            if art_contrib_group.find_all("contrib", recursive=False):
+                contribs_raw_init = art_contrib_group.find_all("contrib", recursive=False)
+
             default_key = "ALLAUTH"
+
+            # cycle through <contrib> to check if a <collab> is listed in the same level as an author an has multiple authors nested under it; targeted for Springer
+            contribs_raw = []
+            collabflag = None
+            for i in range(len(contribs_raw_init)):
+                if contribs_raw_init[i].find("collab"):
+                    # find nested collab authors and unnest them
+                    if contribs_raw_init[i].find("collab").find("institution"):
```

Comment:
Add a comment here that the name of the collab is stored in the institution field - that took me a sec to figure out